scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -22,9 +22,9 @@ task M2 {
   File tumor_bam
```

Comment:
I assume that this has been tested

Please put a note on the cromwell issue that you ahve a workaround.